ansible/install-apps.yml

```diff
@@ -0,0 +1,34 @@
+---
+
+- name: Install OS-level apps
+  hosts: all
+  become: yes
+  vars:
+  - apt_file: /etc/apt/sources.list.d/google-chrome.list
+  - apt_file_Git: 
+  tasks:
+  - name: Does the Google apt file exist?
+    command: test -f {{apt_file}}
+    register: google_apt_exists
+    ignore_errors: yes
+
+  - name: Add Google Chrome key   
+    shell: echo 'deb [arch=amd64] https://dl.google.com/linux/chrome/deb/ stable main'>/etc/apt/sources.list.d/google-chrome.list
+    when: google_apt_exists.rc == 1
```

Comment:
Since modules in Ansible exercise idempotency, you'll see that you won't need to do such checks manually most of the time :)